src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep

```diff
@@ -87,7 +87,7 @@ var not = !'s'
 var not = ![
 //@[04:07) [BCP028 (Error)] Identifier "not" is declared multiple times. Remove or rename the duplicates. (CodeDescription: none) |not|
 //@[04:07) [no-unused-vars (Warning)] Variable "not" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-vars)) |not|
-//@[10:14) [BCP044 (Error)] Cannot apply operator "!" to operand of type "array". (CodeDescription: none) |![\n]|
+//@[10:14) [BCP044 (Error)] Cannot apply operator "!" to operand of type "[]". (CodeDescription: none) |![\n]|
```

Comment:
`[]` is an empty tuple. Maybe this could be represented with a description, such as `[<empty array>]` or just `<empty array>`? The only place we use this notation currently is for `<missing>` or `<error>`, but I think it could work here.